app/views/users/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <div class="text-center">
+    <h1>Your Events</h1>
```

Comment:
Please see my comment on #6

Reply:
hey there is no comment there

Reply:
Sorry haha i meant please revisit #6 :D

Reply:
Hahah ok , we did it like that because a user is meant to visit a user's show page ,but yeah if it better this way we change it right away .Thank you Ryan

Reply:
It is done :)